specification/containerservice/Fleet.Management/clusterresourceplacement.tsp

```diff
@@ -0,0 +1,193 @@
+using TypeSpec.Versioning;
+
+namespace Microsoft.ContainerService;
+
+@added(Versions.v2025_03_02_naar)
+@doc("ClusterResourcePlacement is used to select cluster scoped resources, including built-in resources and custom resources, and placement them onto selected member clusters in a fleet.")
+model ClusterResourcePlacement {
+  @doc("PlacementPolicy defines how to select member clusters to place the selected resources. If unspecified, all the joined member clusters are selected.")
+  policy?: PlacementPolicy;
+}
+
+@added(Versions.v2025_03_02_naar)
+@doc("PlacementPolicy contains the rules to select target member clusters to place the selected resources. Only clusters that are both joined and satisfying the rules will be selected. You can only specify at most one of the two fields: ClusterNames and Affinity.")
+model PlacementPolicy {
+  @doc("Type of placement. Default is PickAll.")
+  placementType?: PlacementType;
```

Comment:
I think Liqian already commented on this, but I wonder why we are making something optional here that is not optional in the source API.

Reply:
Hmm I'm seeing `PlacementType` is optional in the source API:

```go
	// Type of placement. Can be "PickAll", "PickN" or "PickFixed". Default is PickAll.
	// +kubebuilder:validation:Enum=PickAll;PickN;PickFixed
	// +kubebuilder:default=PickAll
	// +optional
	PlacementType PlacementType `json:"placementType,omitempty"`
```
https://github.com/kubefleet-dev/kubefleet/blob/25d684b039ec2b65fd26ddb01afa8048fd6fa6ea/apis/placement/v1/clusterresourceplacement_types.go#L137